azure-pipelines.yml

```diff
@@ -51,16 +51,30 @@ jobs:
 
 - job: 'macos_wheels'
   pool:
-    vmImage: macOS-10.15
+    vmImage: macOS-11
   variables:
     MACOSX_DEPLOYMENT_TARGET: 10.9
     HDF5_VERSION: 1.12.1
+    HDF5_ROS3: '0'
+    HDF5_DIRECT_VFD: '0'
```

Comment:
These 2 environment variables are new and only affect the h5py build, right? It might be clearer to use `H5PY_` as the prefix, to make it clear that they're not controlling the feature in HDF5 itself.

Reply:
Fixed